roles/zuul/templates/etc/zuul/config/layout.yaml

```diff
@@ -67,6 +81,8 @@ projects:
       - ubuntu-hoist
     gate_github:
       - echo-true
+    experimental_github:
+      - ansible-syntax-BonnyCI/hoist
```

Comment:
I agree with jamie that this can go straight into check. I agree with jesuasurus that we will/do need an experimental pipeline.